### ISSUE TITLE
NVDA 2020.4 and Google Sheets in Microsoft Edge

### ISSUE THREAD

**Issue by dTolmachev1** [nvaccess/nvda#12103]
### Steps to reproduce:
Open google sheets and create new spreadsheet in latest Microsoft Edge (88.0). Than try to navigate between cells.
### Actual behavior:
NVDA doesn't read anything, but in 2020.3 everything works fine.
### Expected behavior:
Cell's content reading as usual.
#### NVDA installed/portable/running from source:
Installed.
#### NVDA version:
2020.4.
#### Windows version:
Windows 10 LTSC 2019 (Build 17763.1790).
#### Name and version of other software in use when reproducing the issue:
Microsoft Edge 88.0.705.74.
#### Other information about your system:
I've a lot of addons installed, but I also tried to run NVDA without addons or even clear copy of installation with completely new user configuration -- nothing changed.
#### Does the issue still occur after restarting your computer?
Yes.
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
2020.3 and earlier -- they're work fine.
#### If addons are disabled, is your problem still occuring?
Yes.
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Yes.

**Comment by Adriani90** [comment]
This is not reproducible with NVDA 2022.4 and Microsoft Edge 110. I am closing this as works for me. please comment in case you still have this issue and we can reopen.
